Assets/Character/Scripts/Jump.cs

```diff
@@ -0,0 +1,38 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class Jump : MonoBehaviour {
+    
+    public float moveSpeedY;
+    private bool inJump; 
+    Rigidbody2D rb;
+	// Use this for initialization
+	void Start () {
+        rb = GetComponent<Rigidbody2D>();
+	}
+	
+	// Update is called once per frame
+	void Update () {
+	    if (!inJump)
```

Comment:
I couldn't find any example of someone using get input in Update so just do it in FixedUpdate. Sorry for making the flip flop. So do something more similar to what you had before.